<launch>
   <node pkg="rosbag" type="play" name="rosbag" required="true" args="-d 2 $(find image_view)/Fahrvideo.bag"/>
   <node name="extract" pkg="image_view" type="extract_images" respawn="false" required="true" output="screen" cwd="ROS_HOME">
    <remap from="/image" to="/autonomous/image_raw"/>
  </node>
</launch>
